internal/p2p/peermanager.go

```diff
@@ -101,6 +108,11 @@ type PeerManagerOptions struct {
 	// outbound). 0 means no limit.
 	MaxConnected uint16
 
+	// MaxOutgoingConnections specifies how many outgoing
+	// connections. It must be lower than MaxConnected. If it is
+	// 0, then all connections can be outgoing.
```

Comment:
Can we mention something about Dialing here so that people understand that they are the same